metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -22,7 +22,6 @@
 import uk.ac.ebi.ampt2d.metadata.importer.extractor.FileExtractorFromAnalysis;
```

Comment:
This is a valid concern. I suggest we move it to #55, because I have already performed a significant reordering/grouping of methods and class attributes in that PR. If I change anything here, an unnecessary chain of downstream rebases would be required.

Reply:
We don't order methods as in public, protected then private. We follow the recommendations from the Clean Code book, where a private method follows the first public method that uses it, in order to keep a particular flow as close as possible.

Reply:
Fixed in #55, see [discussion](https://github.com/EBIvariation/metadata-ws/pull/55#pullrequestreview-247590754)